packages/babel-preset-stage-0/README.md

```diff
@@ -1,19 +1,64 @@
 # @babel/preset-stage-0
 
-> Babel preset for stage 0 plugins
+> As of v7.0.0-beta.54, we've decided to remove
+the official Babel Stage presets. You can find more information
+at issue [#7770](https://github.com/babel/babel/issues/7770), but
+the TL;DR is that it's causing more harm than convenience in that
+the preset is always out of date, each change is usually going to
+require a major version bump and thus people will be behind,
+and it encouraged too many people to opt-in to too many proposals
+that they didn't intend to. This is intended to be the last publish
+of "@babel/preset-stage-0"
 
-See our website [@babel/preset-stage-0](https://babeljs.io/docs/en/next/babel-preset-stage-0.html) for more information.
+---
 
-## Install
+If you want the same configuration as before:
 
-Using npm:
+```json
+{
+  "plugins": [
+    // Stage 0
```

Comment:
Comments in JSON are not valid, could we remove them? (because user will need to) or can we switch to js?

Reply:
`.babelrc` uses Json5, so comments work.